tests/calico_testcases/example_policies/expected_output/testcase15_with_ingress_connectivity_map.txt

```diff
@@ -0,0 +1,56 @@
+src_ns: [kube-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [kube-fluentd] conn: TCP {'dst_ports': '80', 'paths': '/xyz/[\\--9A-Z_a-z]+', 'hosts': 'second.bar.com'}
+src_ns: [kube-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [storage-watcher-8494b4b8bb] conn: TCP {'dst_ports': '102', 'paths': '/xyz', 'hosts': 'second.bar.com'}
+src_ns: [kube-system] src_pods: [calico-kube-controllers-7694668c77] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [calico-node] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [file-plugin-7bfb8b69bf] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [heapster-7df8cb8c66] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [kube-dns-amd64-d66bf76db] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [kube-dns-autoscaler-78f5fdbd46] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [kube-fluentd] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [kubernetes-dashboard-5b5f985bcf] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [public-cre08b89c167414305a1afb205d0bd346f-alb1-8489b8458f] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [storage-watcher-8494b4b8bb] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [tiller-deploy-5c45c9966b] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [vpn-858f6d9777] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+
+final fw rules for query: connectivity_map, config: global-simple:
+src_ns: [kube-system] src_pods: [app=keepalived-watcher] dst_ns: [kube-system] dst_pods: [!has(app)] conn: TCP 200-250
+src_ns: [kube-system] src_pods: [app=keepalived-watcher] dst_ns: [vendor-system] dst_pods: [*] conn: TCP 200-250
+
+final fw rules for query: connectivity_map, config: global-simple-with-ingress:
+src_ns: [kube-system] src_pods: [app=keepalived-watcher] dst_ns: [kube-system] dst_pods: [calico-node] conn: TCP {'dst_ports': '210', 'paths': '/abc(/(([\\--9A-Z_abce-z]|d([\\--9A-Z_a-df-z]|e([\\--9A-Z_a-eg-z]|f[\\-.0-9A-Z_a-z])))[\\--9A-Z_a-z]*|d(e(f/)?)?))?', 'hosts': 'first.bar.com'}
+src_ns: [kube-system] src_pods: [app=keepalived-watcher] dst_ns: [kube-system] dst_pods: [kube-dns-amd64-d66bf76db] conn: TCP {'dst_ports': '213', 'paths': '/[\\--9A-Z_a-z]*'}
```

Comment:
Why in this part of the expected output it seems that the Ingress policy affects on allowed connections from a pod which is not the ingress controller?

Reply:
In tests/calico_testcases/example_podlist/service_and_ingress_resources/services_list.yaml this is how I defined ingress controller:
`apiVersion: v1
kind: Service
metadata:
  name: ingress-nginx-controller
  namespace: kube-system
spec:
  type: NodePort
  selector:
    **app.kubernetes.io/name: ingress-nginx**
    **app: keepalived-watcher**
  ports:
    - appProtocol: https
      name: https-webhook
      port: 443
      protocol: TCP
      targetPort: webhook`
So the pods with app: keepalived-watcher are actually the ingress controller pods.